examples/example-vite-react-sdk/src/App.tsx

```
@@ -164,6 +160,7 @@
                                     : "Need to Spawn"}
                             </div>
                             <div className="col-span-3 text-center text-base text-white">
+                                {/* @ts-expect-error we have an option here so type is ok */}
                                 {moves && moves.last_direction.isSome()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)